modules/nw-ovn-kubernetes-rollback.adoc

```diff
@@ -55,17 +55,46 @@ $ oc patch Network.operator.openshift.io cluster --type='merge' \
   --patch '{ "spec": { "migration": null } }'
 ----
 
-. To start the migration, set the network plugin back to OpenShift SDN by entering the following commands:
+. Apply the patch to the `Network.operator.openshift.io` object to set the network plugin back to OpenShift SDN by entering the following command:
 +
 [source,terminal]
 ----
 $ oc patch Network.operator.openshift.io cluster --type='merge' \
   --patch '{ "spec": { "migration": { "networkType": "OpenShiftSDN" } } }'
+----
+
+. Confirm that the `Network.operator.openshift.io cluster` object's network plugin is `OpenShiftSDN` by entering the following command:
++
+[source,terminal]
+----
+$ oc get network.config cluster -o jsonpath='{.status.migration.networkType}'
```

Comment:
Is this correct for the `Network.operator.openshift` object. Should the command state?

oc get network.operator cluster -o jsonpath="{.status.migration.networkType}"

Reply:
No. The `status.migration` field only exists in the network.config CR.

Reply:
Thanks, @pliurh . In the context of the Network.operator.openshift object, what confirmaton command is correct?

Reply:
We don't need to check the status of Network.operator.openshift object here.

Reply:
OK, @pliurh , but as an issue exists with this step (the patch operation is not fully completed), would checking the status be important?

Reply:
The patch operation should be completed immediately upon return. Here, we're verifying that the changes made to the `network.operator` CR have been propagated to the status of the `network.config` CR by the CNO.

Reply:
Thanks. I removed the following content:

. Confirm that the `Network.operator.openshift.io cluster` object's network plugin is `OpenShiftSDN` by entering the following command. If the command output does not show `OpenShiftSDN`, wait a few seconds for the patch operation to complete.
+
[source,terminal,subs="+quotes"]
----
$ oc get Network.config cluster -o jsonpath='{.status.migration.networkType}'
----

Reply:
@pliurh are we okay to remove this ^? We don't want user to wait until the patch command is successful?

Reply:
We do want users to wait until the patch command is successful. Users can know that if the patch command fails from the return message.

Reply:
Thanks, @pliurh . @Rupesh-git-eng , the output would signify an issue so we do not need to add any customer intervention to check the status.